src/fe/fe_map.C

```diff
@@ -383,8 +383,23 @@ void FEMap::compute_single_point_map(const unsigned int dim,
 
         if (jac[p] <= 0.)
           {
-            elem->print_info();
-            libmesh_error_msg("ERROR: negative Jacobian: " << jac[p] << " in element " << elem->id());
+            // Don't call print_info() recursively if we're already
+            // failing.  print_info() calls Elem::volume() which may
+            // call FE::reinit() and trigger the same failure again.
+            static bool failing = false;
+            if (!failing)
+              {
+                failing = true;
+                elem->print_info();
+                libmesh_error_msg("ERROR: negative Jacobian " << jac[p] << " at point " << xyz[p] << " in element " << elem->id());
```

Comment:
OK, but only if we also follow [these](http://web.stanford.edu/class/me200c/tutorial_77/03_basics.html) guidelines...
```
Col. 1    : Blank, or a "c" or "*" for comments
Col. 1-5  : Statement label (optional)
Col. 6    : Continuation of previous line (optional)
Col. 7-72 : Statements
Col. 73-80: Sequence number (optional, rarely used today)
```